site/contents/write_ups/2022/12_01_22-writeup.md

```diff
@@ -0,0 +1,93 @@
+
+Ola will be trying to take a slightly more positive outlook on things whilst still being thorough. Paper reviewers could benefit from this too! This idea resonated with lots of members of the group, who missed the days when they could genuinely read for joy instead of just for work. Unfortunately PhD students are just being optimised for picking out "what can I get out of this?" from any source they read from
+
+Ola also mentioned the [civic labratory](https://civiclaboratory.nl/2021/01/03/collabrary-a-methodological-experiment-for-reading-with-reciprocity/), a group asking themselves: "How can we change academic reading relations that tend to be extractive into something more reciprocal, humble, generous, and accountable?"
+
+This led us to asking if there is a dichotomy between being nice and being pessimistic? You don't always have to read to evaluate, unless you're doing a systematic literature review. You can just read, as if it was a fiction book. Maybe asking different questions might make it better. Can you read for how the author arrived at arguments? What claims did they make? What were they supported by?
+
+This is not just about being a nicer person. If your own wellbeing suffers from being a bit salty, maybe you need to learn to enjoy reading again. Film snobs tend to enjoy films less, maybe we reading snobs should learn to enjoy reading again.
+
+## Blowing the Whistle and Looking at Jobs
+
+One of our members, Euan has intents to "blow the whistle" this year in his field. His studies are not being listened to, even though it would make things safer in his field. Euan intends to try and continue to look for opportunities to apply more of what we learn at Data Ethics Club to what he is doing.
+
+This also could mean examining exit routes out of academia. A lot of the career paths in academic are not looking as appealing, even if the actual work is good. Is there a way to integrate ethical behaivour into data science consultancy, perhaps even making it part of the brand? Stay tuned!
+
+Heather also is considering a job change, but is struggling with finding job oppurtunities in areas she considers ethical. In the future at data ethics club we hope to have a discussion on ethical job hunting and what it means for a job to be ethical. Roman also wants to stay responsible when entering the job market as he finishes his degree.
+
+Ben wants to be braver about calling behaviours out as well as being more prescriptive (should/must) because its morally wrong to do the other option (sit back and do nothing). Mia wants to be more willing to have difficult conversations with colleagues and be more comfortable in those uncomfortable discussions. Beverly hopes to build a stronger community and start to use their voice more.
```

Comment:
```suggestion
Ben wants to be braver about calling behaviours out as well as being more prescriptive (should/must) because its morally wrong to do the other option (sit back and do nothing). 

Similarly Mia wants to be more willing to have difficult conversations with colleagues and be more comfortable in those uncomfortable discussions. 

Beverly hopes to build a stronger community and start to use their voice more to affect change in areas that matter.
```